xtask/src/commands/release.rs

```diff
@@ -58,6 +64,11 @@ pub enum Release {
     /// - Update the version in the changelog
     /// - Replaces `{{currentVersion}}` markers in source files and the
     ///   migration guide.
+    #[cfg(feature = "release")]
+    /// Rollover migrations steps post release.
+    /// - Create new migration guides for packages that have a migration guide
+    PostRelease,
```

Comment:
I'm not sure why you want to have BumpVersion's documentation on top of PostRelease 🤔

Reply:
Oops! Not sure what happened here, will fix.